tests/phpunit/tests/block-template.php

```diff
@@ -275,6 +275,82 @@ public function test_get_the_block_template_html_does_not_generally_enforce_loop
 		$this->assertSame( array( false, true ), $in_the_loop_logs, 'Main query loop was triggered incorrectly' );
 	}
 
+	/**
+	 * @ticket 58319
+	 *
+	 * @covers ::get_block_theme_folders
+	 */
+	public function test_get_block_theme_folders_block_theme() {
```

Comment:
We should use a data provider here, so we can check different themes.

Reply:
Thanks @spacedmonkey  for this suggestion. I have created a dataProvider and added few more cases to it.